hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -853,6 +853,11 @@ public static boolean isAclEnabled(Configuration conf) {
   /** Zookeeper interaction configs */
   public static final String RM_ZK_PREFIX = RM_PREFIX + "zk-";
 
+  /** Enable Zookeeper SSL/TLS communication */
+  public static final String RM_ZK_CLIENT_SSL_ENABLED =
+      RM_ZK_PREFIX + "client-ssl.enabled";
+  public static final boolean DEFAULT_RM_ZK_CLIENT_SSL_ENABLED = Boolean.FALSE;
```

Comment:
In this case I don't think it makes too much sense to use the static Boolean.FALSE object as the type is a primitive boolean anyways. 
See: https://stackoverflow.com/questions/20090306/what-is-the-difference-between-false-and-boolean-false